source/includes/_contributing.mdown

```diff
@@ -1,32 +1,41 @@
-# Getting help
+# Contributing
 
-## Discord
+Want to hack on Origin? Awesome!
 
-Our team collaboration is done in public and our company Discord is open to all. If you have questions or need help getting started, our [Discord #engineering channel](https://www.originprotocol.com/discord) is a great place to get assistance from our team of engineers and developers.
+Origin is an Open Source project and we welcome contributions of all sorts. There are many ways to help, from reporting issues, contributing code, and helping us improve our community.
 
-## Email
+There are four main repositories:
```

Comment:
Many of the "good first issues" are actually on the company-website repo, or even this one. We may or may not want to steer people toward the product-related repos (the four mentioned here). For that matter, "good first issues" are generally fairly simple and _some_ of our target audience (blockchain and advanced developers) might be more interested in tackling a more "interesting" issue. Maybe we suggest that people search the issues by programming language or another label best done [here](https://github.com/OriginProtocol).

Reply:
This documentation is really focused on the 3 platform-related repos. 

We should probably separate the identity playground, but let's mention the other repos like the company website as other places that are sometimes better to get started.